test/unittest/utils/LocatorTests.cpp

```diff
@@ -1841,26 +1841,26 @@ TEST(LocatorDNSTests, resolve_name)
 }
 
 /*
-    This test uses the DNS and IPs onf the `resolve_name` test
+    This test uses the DNS and IPs of the `resolve_name` test
```

Comment:
If the same values should be used for several tests, we could define them once at the beginning of the file.